go-controller/pkg/ovn/namespace.go

```diff
@@ -47,11 +48,8 @@ func (oc *Controller) addPodToNamespace(ns string, portInfo *lpInfo) error {
 	}
 	defer nsInfo.Unlock()
 
-	//FIXME DUAL-STACK
-	if len(portInfo.ips) > 0 {
-		if err := nsInfo.addressSet.AddIP(portInfo.ips[0].IP); err != nil {
-			return err
-		}
+	if err := nsInfo.addressSet.AddIPs(createIPAddressSlice(portInfo.ips)); err != nil {
```

Comment:
The need for `createIPAddressSlice` here suggests that maybe it would have been better to keep `AddIP` rather than `AddIPs` in `AddressSet`, and do the looping in the caller rather than there?

Reply:
Actually `AddIPs` was needed everywhere. For example this still makes perfect sense in `addPeerPod` and `deletePeerPod` methods which uses AddIPS/.DeleteIPs method without doing any manipulation on passed in data.  But one of the PRs refactored the `Portinfo` data structure  and changed the type of ips to  []*net.IPNet. Hence when I rebased the code, I had to create this method.